README.rst

```diff
@@ -133,6 +133,62 @@ input queries before sending them to the API:
 Run ``python -m autoextract --help`` to get description of all supported
 options.
 
+Errors
+~~~~~~
+
+The following errors could happen while making requests:
+
+- Network errors
+- `Request-level errors`_
+    - Authentication failure
+    - Malformed request
+    - Too many queries in request
+    - Request payload size is too large
+- `Query-level errors`_
+    - Downloader errors
+    - Proxy errors
+    - ...
+
+Some errors could be retried while others don't.
```

Comment:
Some errors _can_ be..

Reply:
Both are modal verbs and I think they have very similar meaning despite slightly different intonation in this case. I'm replacing it anyway because I think it makes the sentence sound more clear and certain.